tests/test_format_strings.py

```diff
@@ -1,5 +1,6 @@
 import pytest
 
+from . import check
 from vulture import Vulture
```

Comment:
```
import pytest
from vulture import Vulture

from . import check
```
We should treat vulture as any other non-stdlib module. (This applies to the other test files as well)